.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,11 +9,15 @@ assignees: ''
 
 ---
 
+Download and run checkstyle cli: https://checkstyle.org/cmdline.html#Download_and_Run
+
 Check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck
 
+Cli describes the problem better than 1,000 words.
```

Comment:
idea on how to make this wording as it would be written by each issue author:
```
I read Check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck
I downloaded checkstyle from https://checkstyle.org/cmdline.html#Download_and_Run 
and show how problem by CLI execution because CLI describes the problem better than 1,000 words
```

Reply:
This is a good idea.

Reply:
Done